.pre-commit-config.yaml

```diff
@@ -40,3 +40,4 @@ repos:
           - "types-requests"
           - "sqlalchemy2-stubs"
           - "types-attrs"
+          - "types-toml"
```

Comment:
now not needed anymore since we would not be using toml.
But it was needed because of [failing workflow.](https://app.circleci.com/pipelines/github/up42/up42-py/3671/workflows/f6516398-49fd-4ff3-b683-39371e4a2f2a/jobs/4357)